remoulade/state/backend.py

```diff
@@ -47,18 +47,17 @@ class State(
     def __new__(
         cls,
         message_id,
-        name,
+        name=None,
         *,
         actor_name=None,
         args=None,
         kwargs=None,
         priority=None,
+        progress=None,
         enqueued_datetime=None,
         started_datetime=None,
         end_datetime=None
     ):
-        if name not in list(StateNamesEnum):
```

Comment:
You should not delete this, but check it only if name is not none (update changelog)